fastlane/Fastfile

```diff
@@ -38,8 +38,8 @@ platform :ios do
     setup_circle_ci
   end
 
-  desc "Deploy to MS App-Center"
-  lane :appcenter do
+  desc "Deploy to MS App-Center + Browserstack"
+  lane :upload_to_appcenter_and_browserstack do
```

Comment:
Or maybe at least could be two different fastlane actions to avoid this `and` pattern and leaving a single responsibility.